src/main/java/org/example/lesson23/TomCatPractice/src/main/java/com/example/tomcatpractice/service/OrderRepository.java

```diff
@@ -0,0 +1,29 @@
+package com.example.tomcatpractice.service;
+
+import com.example.tomcatpractice.dao.OrderDao;
+import com.example.tomcatpractice.entity.Order;
+import com.example.tomcatpractice.exception.OrderNotFoundException;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class OrderRepository implements Repository<Order> {
```

Comment:
Please, use Service instead of Repository
You already used a layer to communicate with DB it's a DAO